x-pack/plugins/siem/public/management/pages/policy/view/policy_list.tsx

```diff
@@ -89,63 +95,110 @@ export const PolicyList = React.memo(() => {
   const columns: Array<EuiTableFieldDataColumnType<Immutable<PolicyData>>> = useMemo(
     () => [
       {
-        field: 'name',
+        field: '',
```

Comment:
:grey_question: Why did you set this to an empty string? Shoudn't the column view have a 'field' present?

Reply:
The `field` actually points to a field in the response in the `EuiBasicTable` component. In this case I just want the top level `item` (i.e. the entire object in state from the API response).  @paul-tavares @parkiino not sure if there is a cleaner way to get the whole object?

Docs: https://elastic.github.io/eui/#/tabular-content/tables

Reply:
@kevinlog you always get the `item` as the second argument to `render()` callback, so you could:

```typescript
{
  render: (_, item ) => (/* item === row object record */)
}
```